src/Orleans.Core/Runtime/Constants.cs

```diff
@@ -31,6 +31,7 @@ internal class Constants
         public static readonly GrainType DirectoryCacheValidatorType = SystemTargetGrainId.CreateGrainType("dir.cache-validator");
         public static readonly GrainType SiloControlType = SystemTargetGrainId.CreateGrainType("silo-control");
         public static readonly GrainType ClientObserverRegistrarType = SystemTargetGrainId.CreateGrainType("observer.registrar");
+        public static readonly GrainType ClientDirectoryType = SystemTargetGrainId.CreateGrainType("client-dir");
```

Comment:
Out of curiosity, why `.`, dot, in some constants and `-`, a hyphen, in some?

Reply:
The original idea was to separate the category ("dir" "txn") from the component/implementation ("agent" "mem" "cache-validator"), but they appear inconsistent.